### PR TITLE
feat: register multiple commands

### PR DESCRIPTION
**Pull request by DebitCardz**
Makes it easier to register multiple commands at once.

### CODE REVIEW

**Comment by mudkipdev** [comment]
I think it would be better if you added the vararg to the main register method instead of making an overload.

**Comment by DebitCardz** [comment]
> I think it would be better if you added the vararg to the main register method instead of making an overload.
Maybe, but I really don't want to potentially mess with someones reflection garbage for a simple QoL change.

File: src/main/java/net/minestom/server/command/CommandManager.java [hunk 1]
```diff
@@ -60,6 +60,18 @@ public synchronized void register(@NotNull Command command) {
         invalidateGraphCache();
     }
 
+    /**
+     * Register multiple {@link Command}.
```
Comment (mudkipdev) [inline src/main/java/net/minestom/server/command/CommandManager.java:64]: Should be plural "commands".

Comment (DebitCardz) [reply src/main/java/net/minestom/server/command/CommandManager.java:64]: Links to the actual Command class, would you prefer `Commands` or `{@link Command}s`?

Comment (mworzala) [reply src/main/java/net/minestom/server/command/CommandManager.java:64]: the second one

File: src/main/java/net/minestom/server/command/CommandManager.java [hunk 2]
```diff
@@ -60,6 +60,18 @@ public synchronized void register(@NotNull Command command) {
...
+     *
+     * @param commands the array of commands
+     * @throws IllegalStateException if a command with the same name already exists
+     */
+    public synchronized void register(@NotNull Command ...commands) {
```
Comment (mworzala) [inline src/main/java/net/minestom/server/command/CommandManager.java:69]: ... should be before the space, and there should be a space after `for`

**Comment by mworzala** [comment]
im fine with the overload

**Pull request merged**